csi/src/mayastor_svc.rs

```
@@ -307,8 +307,21 @@ impl service::mayastor_server::Mayastor for MayastorService {
     ) -> Result<Response<Null>, Status> {
         let msg = request.into_inner();
         trace!("{:?}", msg);
-        jsonrpc::call::<_, ()>(&self.socket, "offline_child", Some(msg))
-            .await?;
+        match msg.action {
```

Could we add a helper child action to string? And maybe we return an error is it's an invalid action?